packages/atomic/src/components/commerce/product-template-components/atomic-product-image/atomic-product-image.new.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import {wrapInCommerceInterface} from '@coveo/atomic-storybook-utils/commerce/commerce-interface-wrapper';
+import {wrapInCommerceProductList} from '@coveo/atomic-storybook-utils/commerce/commerce-product-list-wrapper';
+import {wrapInProductTemplate} from '@coveo/atomic-storybook-utils/commerce/commerce-product-template-wrapper';
+import {parameters} from '@coveo/atomic-storybook-utils/common/common-meta-parameters';
+import {renderComponent} from '@coveo/atomic-storybook-utils/common/render-component';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import type {Decorator} from '@storybook/web-components';
+import {html} from 'lit-html';
+
+const styledDivDecorator: Decorator = (story) => {
+  return html`<div style="max-width: 700px">${story()}</div>`;
+};
+
+const {
+  decorator: commerceInterfaceDecorator,
+  play: initializeCommerceInterface,
+} = wrapInCommerceInterface({
+  skipFirstSearch: false,
+  type: 'product-listing',
+  engineConfig: {
+    context: {
+      view: {
+        url: 'https://ui-kit.coveo/atomic/storybook/atomic-product-image',
+      },
+      language: 'en',
+      country: 'US',
+      currency: 'USD',
+    },
+  },
+});
+const {decorator: commerceProductListDecorator} = wrapInCommerceProductList();
+const {decorator: productTemplateDecorator} = wrapInProductTemplate();
+
+const meta: Meta = {
+  component: 'atomic-product-image',
+  title: 'Atomic-Commerce/Product Template Components/ProductImage',
+  id: 'atomic-product-image',
+  render: renderComponent,
+  decorators: [
+    productTemplateDecorator,
+    commerceProductListDecorator,
+    commerceInterfaceDecorator,
+    styledDivDecorator,
+  ],
+  parameters,
+  play: initializeCommerceInterface,
+};
+
+export default meta;
+
+export const Default: Story = {
+  name: 'atomic-product-image',
+};
+
+export const withAFallbackImage: Story = {
+  name: 'With a fallback image',
+  args: {
+    'attributes-fallback': 'https://sports.barca.group/logos/barca.svg',
+  },
+};
+
+export const withAnAltTextField: Story = {
+  tags: ['test'],
+  name: 'With an alt text field',
+  args: {
+    'attributes-image-alt-field': 'custom_alt_field',
+  },
```

Comment:
The default one is a hardcoded text : "Image {index} out of {totalImages} for {productName}"

Reply:
There is no default `image-alt-field`. I think `custom_alt_field` shows clearly what it should look like.

Reply:
It does show what the name of a custom field would look like, but it's using an invalid field so that's not great :D 

One possible solution would be to use ec_name or maybe ec_product_id as an alt field in the story.

Reply:
Oh okay, I see I thought the point of `image-alt-field` was to create your own field for that matter not necessarily reusing one.  I can change it to ec_name right

Reply:
changd it to ec_name

Reply:
Instead of changing it, I tagged the other story as test and made another one for the showcase. Changing it to ec_name broke too much of my tests because ec_name is never invalid.

<img width="605" alt="image" src="https://github.com/user-attachments/assets/9c291f4f-9881-4336-bf0b-0a9964c93883">